docs/install.md

```diff
@@ -1,26 +1,28 @@
 # Installing Calypso
 
+You can install Calypso directly on your machine by following the next steps, or use a [portable development environment](install.md#using-a-portable-development-environment):
+
 ## Quick Summary of Steps
 
-1.	Check that you have all prerequisites (Git, Node, NPM). See below for more details.
+1.	Check that you have all prerequisites (Git, Node, NPM). See [below](install.md#prerequisites) for more details.
 2.	Clone this repository locally.
-3.	Add `127.0.0.1 calypso.localhost` to your local hosts file.
-4.	Execute `make run` from the directory of the repository.
-5.	Open `calypso.localhost:3000` in your browser.
+3.	Add `127.0.0.1 calypso.localhost` to your local `hosts` file.
+4.	Execute `make run` or `make dashboard` (for a more visually-oriented interface) from the root directory of the repository.
+5.	Open [`calypso.localhost:3000`](http://calypso.localhost:3000/) in your browser.
 
 ## Prerequisites
 
-To be able to clone the repo and run the application **on Mac OS X or Linux** you need:
+**On Mac OS X or Linux**, to be able to clone the repository and run the application you need:
 
 -	Install the [Node.js](http://nodejs.org/) and matching [NPM](https://www.npmjs.com/) [versions](https://nodejs.org/en/download/releases/) listed in the engines section of [package.json](https://github.com/Automattic/wp-calypso/blob/master/package.json). On Mac OS X using [nvm](https://github.com/creationix/nvm) or [n](https://github.com/tj/n) makes it easy to manage `node` versions.
 -	[Git](http://git-scm.com/). Try the `git` command from your terminal, if it's not found then use this [installer](http://git-scm.com/download/).
 -	The repository also uses `make` to orchestrate compiling the JavaScript, running the server, and several other tasks. On Mac OS X, the easiest way to install `make` is through Apple's [Command Line Tools for Xcode](https://developer.apple.com/downloads/) (requires free registration).
 
-**On Windows**, please follow [these setup steps](windows.md) to install the prerequisites, or see the [alternate install](install.md#alternate-install-vagrant) instructions further down.
+**On Windows**, you can follow [these setup steps](windows.md) to install the prerequisites.
```

Comment:
I suspect so as well. However I removed this part because this option is now mentioned in the first paragraph of this document, although without targeting Windows users explicitly. I'll update [the document](https://github.com/Automattic/wp-calypso/blob/master/docs/windows.md) linked here to mention Calypso Bootstrap as an alternative to make sure this is something that users don't miss then.